hw/top_earlgrey/data/ip/chip_pwrmgr_testplan.hjson

```diff
@@ -0,0 +1,407 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_pwrmgr
+  testpoints: [
+    // PWRMGR tests:
+    {
+      name: chip_sw_pwrmgr_external_full_reset
+      desc: '''Verify the cold boot sequence by wiggling of chip's `POR_N`.
+
+            This ensures that both FSMs are properly reset on the POR signal. The check  is that
+            the processor ends up running. Also verify, the rstmgr recorded POR in `reset_info` CSR
+            by checking retention SRAM for reset_reason.
+            SiVal: This will be exercised without any special test, just by using the external POR.
+            '''
+      stage: V2
+      si_stage: SV1
+      lc_states: [
+        "RAW",
+        "TEST_LOCKED",
+        "TEST_UNLOCKED",
+      ]
+      features: ["PWRMGR.RESET.POR_REQUEST"]
+      tests: ["chip_sw_pwrmgr_full_aon_reset"]
```

Comment:
need a `bazel: []` field here (applies throughout where `si_stage: None` is not present)

Reply:
Got it, done.